src/test/java/net/snowflake/client/jdbc/ResultSetMultiTimeZoneLatestIT.java

```diff
@@ -152,21 +142,32 @@ public void testSessionTimezoneUsage() throws SQLException {
   @Test
   @ConditionalIgnoreRule.ConditionalIgnore(condition = RunningOnGithubAction.class)
   public void testUseSessionTimeZoneOverrides() throws SQLException {
-    helperTestUseSessionTimeZone(true, false);
+    testUseSessionTimeZoneHelper(false);
   }
 
-  private void helperTestUseSessionTimeZone(
-      boolean useSessionTimezone, boolean useDefaultParamSettings) throws SQLException {
+  /**
+   * Helper function to test behavior of parameter JDBC_USE_SESSION_TIMEZONE. When
+   * JDBC_USE_SESSION_TIMEZONE=true, time/date/timestamp values are displayed using the session
+   * timezone, not the JVM timezone. There should be no offset between the inserted value and the
+   * displayed value from ResultSet. For example, a timestamp value inserted as: 2019-01-01
+   * 17:17:17.6 +0500 will get displayed as so: ResultSet.getTimestamp(): 2019-01-01 17:17:17.6;
+   * ResultSet.getTime(): 17:17:17.6; ResultSet.getDate(): 2019-01-01
+   *
+   * <p>When JDBC_USE_SESSION_TIMEZONE=false, the displayed values will be different depending on
+   * the timezone offset between the session and JVM timezones.
+   *
+   * @param useDefaultParamSettings use default settings of other time/date session formatting
+   *     parameters
+   * @throws SQLException
+   */
+  private void testUseSessionTimeZoneHelper(boolean useDefaultParamSettings) throws SQLException {
     Connection connection = init();
     Statement statement = connection.createStatement();
     // create table with all timestamp types, time, and date
     statement.execute(
         "create or replace table datetimetypes(colA timestamp_ltz, colB timestamp_ntz, colC timestamp_tz, colD time, colE date)");
-    if (useSessionTimezone) {
-      statement.execute("alter session set JDBC_USE_SESSION_TIMEZONE=true");
-    } else {
-      statement.execute("alter session set JDBC_USE_SESSION_TIMEZONE=false");
-    }
+    // Enable session parameter JDBC_USE_SESSION_TIMEZONE
+    statement.execute("alter session set JDBC_USE_SESSION_TIMEZONE=true");
```

Comment:
Do we care about the code coverage for `JDBC_USE_SESSION_TIMEZONE = false`?

Reply:
No, it was more of a sanity check to make sure the parameter didn't alter existing behavior when turned off. I thought about it and realized I can remove it because JDBC_USE_SESSION_TIMEZONE is false by default and we have a lot of other code coverage on date/time formatting elsewhere ResultSetMultiTimeZoneIT, ResultSetIT, ResultSetJsonVsArrowIT, etc, all which test with JDBC_USE_SESSION_TIMEZONE=false.

Reply:
I found this https://docs.snowflake.com/en/sql-reference/parameters.html#jdbc-use-session-timezone so it is a public parameter. So I think we still need to keep support it so I'm worried that we are losing the coverage after this PR.

Reply:
Yes, but these tests are meant to test the behavior of when JDBC_USE_SESSION_TIMEZONE = true. All other date/time tests already test when JDBC_USE_SESSION_TIMEZONE = false because it is false by default.